### PR TITLE
feat: update sentiment meter backend integration

### PR DESCRIPTION
**Pull request by bharathraj25**
* Updated sentimeter endpoints acc to requirments needed on frontend.
* Handled user based access

### CODE REVIEW

File: prisma/dashboard-db/schema.prisma [hunk 1]
```diff
@@ -147,6 +146,7 @@ enum Type {
 }
 
 enum Sentiment {
+  NULL
```
Comment (sugumar-mani) [inline prisma/dashboard-db/schema.prisma:149]: Why this entry?

File: src/services/dsm/sentimentMeter.services.js [hunk 2]
```diff
@@ -132,11 +234,40 @@ const deleteSentimentById = async (sentimentId, memberId, projectId) => {
...
+      sentiment,
+      projectId,
+      memberId,
+    },
+    ...selectOnlyValidSentimentMeterFields,
+  });
+  return newSentiment;
+};
+
+
```
Comment (sugumar-mani) [inline src/services/dsm/sentimentMeter.services.js:264]: remove extra space

File: prisma/dashboard-db/schema.prisma (hunk 1 above)
Comment (bharathraj25) [reply prisma/dashboard-db/schema.prisma:149]: Got it, will be updating

File: src/services/dsm/sentimentMeter.services.js (hunk 2 above)
Comment (bharathraj25) [reply src/services/dsm/sentimentMeter.services.js:264]: done

**Review by samim-hossain-mondal — APPROVED** [review]
LGTM !

**Pull request merged**